source/cpu/rdt_amd64.go

```diff
@@ -75,17 +83,50 @@ func discoverRDT() []string {
 	if extFeatures.EBX&EXT_FEATURE_FLAGS_EBX_RDT_A != 0 {
 		// L3 Cache Allocation
 		if rdtAllocation.EBX&RDT_ALLOCATION_EBX_L3_CACHE_ALLOCATION != 0 {
-			features = append(features, "RDTL3CA")
+			attributes["RDTL3CA"] = "true"
+			numClosID := getNumClosID("L3")
+			if numClosID > -1 {
+				attributes["RDTL3CA_NUM_CLOSID"] = strconv.FormatInt(int64(numClosID), 10)
+			}
 		}
 		// L2 Cache Allocation
 		if rdtAllocation.EBX&RDT_ALLOCATION_EBX_L2_CACHE_ALLOCATION != 0 {
-			features = append(features, "RDTL2CA")
+			attributes["RDTL2CA"] = "true"
 		}
 		// Memory Bandwidth Allocation
 		if rdtAllocation.EBX&RDT_ALLOCATION_EBX_MEMORY_BANDWIDTH_ALLOCATION != 0 {
-			features = append(features, "RDTMBA")
+			attributes["RDTMBA"] = "true"
 		}
 	}
 
-	return features
+	return attributes
+}
+
+func getNumClosID(level string) int64 {
+	resctrlRootDir, err := intelrdt.Root()
+	if err != nil {
```

Comment:
Should we log this as well, WDYT?

```suggestion
	if err != nil {
		klog.V(4).ErrorS(err, "failed to determine resctrlfs root directory")
```

Reply:
Yeah, missed one line.

Reply:
fixed